External/SPMmodified/xASL/xASL_im_FillNaNs.m

```diff
@@ -73,18 +70,23 @@
     bQuality = 1; % default
 end
 
-if UseMethod==1 % only method that needs VoxelSize
-    if nargin<4 || isempty(VoxelSize)
+if UseMethod == 1 || UseMethod == 3 % only methods that needs VoxelSize
+    if nargin < 4 || isempty(VoxelSize)
         if ~isnumeric(InputPath) && xASL_exist(InputPath, 'file')
             nii = xASL_io_ReadNifti(InputPath);
             VoxelSize = nii.hdr.pixdim(2:4);
+		elseif UseMethod == 3
```

Comment:
See above

Reply:
See above - it will estimate the Voxel-size automatically from the deformation fields. So it will work as intended.

Reply:
Just remove VoxelSize as input parameter to METHOD 3 and keep it for METHOD 1, would be my preference.

Reply:
OK. will do